### DISCUSSION TITLE
change the position of TButton in a TWindow

### DISCUSSION THREAD

**Discussion by paule32** [magiblot/tvision#127] [Q&A] [answered]
based on TVDemo.cpp, I have assign cmZoom to toogleZoom (from StatusBar).
The methode will be execute, but I have the problem, that the TButton (button1, and button2)
will be disappear.
My goal is it, on each zoom of a TWindow, the TButton's will be update on the right side of the TWindow (TDialog).
```
void toogleZoom()
{
    TRect R = getExtent();
    
    button1->changeBounds( TRect( R.a.x - 19, 2, R.a.x - 4, 4 ) );
    button2->changeBounds( TRect( R.a.x - 19, 4, R.a.x - 4, 6 ) );
    
    drawView();
}
```
Thanks for reading
paule32

**Comment by magiblot** [comment]
You need to change the `growMode` property of your buttons. It looks like you expect the buttons to behave the same as the ones in `TFileDialog`. Therefore, the values you need are:

https://github.com/magiblot/tvision/blob/f369a73d835c010fc96f3fae505c73766157d13f/source/tvision/tfildlg.cpp#L93

**Reply by magiblot** [reply, marked as answer]
You may also have to change the `growMode` of the editor view (which I understand is on the left side of the dialog). I guess the value you need is `gfGrowHiX | gfGrowHiY`.
